qpc/insights/__init__.py

```diff
@@ -15,7 +15,7 @@
 UPLOAD = 'upload'
 
 REPORT_URI = '/api/v1/reports/'
-DEPLOYMENTS_PATH_SUFFIX = '/deployments/'
+DEPLOYMENTS_PATH_SUFFIX = '/insights/'
```

Comment:
should this still be called `DEPLOYMENTS_PATH_SUFFIX`?

Reply:
yep